frontend/src/static/js/v_zoom.js

```diff
@@ -60,12 +64,16 @@ window.vZoom = {
           .length;
     },
   },
-  created() {
-    this.filterCommits();
-  },
   mounted() {
+    this.filterCommits();
     this.updateExpandedCommitMessagesCount();
   },
+  watch: {
```

Comment:
Can move the `watch` methods to be below the `data()`? To standardise the position of Vue hooks.